### ISSUE TITLE
Gboard swipe disabled

### ISSUE THREAD

**Issue by pjwarner** [connectbot/connectbot#685]
## Bug description

When using gboard in connectbot I am unable to use either swipe typing or suggestion typing.
## Steps to reproduce

Open ConnectBot with the Gboard keboard enabled
Attemp to type using swipe/slide gestures

## Expected behavior

Typing should occur, but only selection of last key is completed
## Android device

 - Device: Google Pixel 1
 - OS: Android 9
 - ConnectBot Version: 1.9.5

## Additional context

Basically I just expect that I would be able to use the swipe function as typing with my fingers is cumbersome.

**Comment by BlueH2O** [comment]
I concur, this is frustrating.

**Comment by BonesGit** [comment]
Would love this fixed.
